rules/opentitan/exec_env.bzl

```diff
@@ -32,6 +32,8 @@ _FIELDS = {
     "otp_data_perm": ("attr.otp_data_perm", False),
     "flash_scramble_tool": ("attr.flash_scramble_tool", False),
     "rom_scramble_config": ("file.rom_scramble_config", False),
+    "openocd": ("attr.openocd", False),
+    "openocd_adapter_config": ("attr.openocd_adapter_config", False),
```

Comment:
Should this be `None`?

Reply:
The comment above says
```starlark
# ExecEnvInfo provider fields and whether the field is required.
```
and this is only used for boolean testing. The actual value is retrieved from ctx.